torch/onnx/symbolic_opset9.py

```diff
@@ -6431,8 +6389,7 @@ def prim_unchecked_cast(g: jit_utils.GraphContext, self):
 def prim_dtype(g: jit_utils.GraphContext, self):
     scalar_name = symbolic_helper._try_get_scalar_type(self)
     if scalar_name is None:
-        scalar_name = "Float"
-    scalar_type = _type_utils.JitScalarType.from_name(scalar_name)
+        scalar_name = _type_utils.JitScalarType.FLOAT
```

Comment:
scalar_name is not used. Is this intended?

Reply:
nop, incomplete renaming